src/app/routes/applications/new-staking-pool/new-staking-pool.component.scss

```diff
@@ -42,6 +44,97 @@
   .staking-date-picker.mat-form-field:not(.mat-form-field-appearance-legacy) .mat-form-field-suffix .mat-icon-button {
     background-color: transparent;
   }
+
+  .editor {
+    border: 1px solid rgb(82, 67, 110);
+
+    &:focus-within {
+      border: 1px solid rgb(103, 58, 183);
+    }
+
+    .NgxEditor {
+      border: none;
+      border-radius: 0;
+      background: #1C1726;
+      color: white;
+      height: $editor-height;
+      overflow-y: scroll;
+
+      &__Popup {
```

Comment:
Do you like styling this way?

Reply:
maybe instead of showing the URL pop up below. It will show on top?